tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/functions/CustomFunctionTest.java

```diff
@@ -43,4 +43,12 @@ public void testDuplicateParameters() {
         BAssertUtil.validateError(compile, 0, "redeclared symbol 'param'", 1, 35);
     }
 
+    @Test(description = "Test defining ballerina function with extra parameters with missing comma")
+    public void testExtraParameters() {
```

Comment:
I'm not sure if the name/description is accurate?

Reply:
The issue is when an array-type-descriptor is used as an expression right? Not an issue with parameters?

For example, even the following fails

```ballerina
function foo(any a) returns int {
    return 1;
}

typedesc t = byte[];

int a = foo(t);
```

Shall we rename and move this test to `TypedescTests` maybe?

Reply:
+1 will move